src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -784,7 +787,8 @@ private static boolean isZeroDistanceToken(int type) {
                 || type == TokenTypes.MODIFIERS
                 || type == TokenTypes.RESOURCE
                 || type == TokenTypes.EXTENDS_CLAUSE
-                || type == TokenTypes.IMPLEMENTS_CLAUSE;
+                || type == TokenTypes.IMPLEMENTS_CLAUSE
+                || type == TokenTypes.CLASS_DEF;
```

Comment:
> removed conditional - replaced equality check with true

This is my local when I run the profile.
![Image](https://github.com/checkstyle/checkstyle/assets/5427943/10eade46-e48f-4c79-9b0d-d4d702aff5cf)

I removed one of these lines completely, or something similar to changing `|| type == TokenTypes.EXTENDS_CLAUSE` to `|| false`, and all the tests passed. This shows the mutation. The mutation in this case is reversed from the text that says true.

So now we must identify how to kill these mutations, or understand why they can't be killed. I would look over https://github.com/checkstyle/checkstyle/wiki/How-to-Generate-and-Analyze-Pitest-Reports#how-to-find-java-code-that-can-kill-mutation and https://github.com/checkstyle/contribution/tree/master/checkstyle-tester#checkstyle-pitest-regression for the next steps. You can also look over other pitest issues that describe the more complex process. We do not normally accept new code with new surviving mutations.

Reply:
>At a glance this sort of makes sense. extends and implements both have class as a parent.

Yes, this sounds like what is happening here. We should probably add some test cases for local enums, records, and interfaces as well.

Reply:
Thank u so much!!! I see. Trying to solve it.